src/backend/aspen/tests/test_backend_serves_requests.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+
+def test_backend_serves_requests():
+
+    backend_request = requests.get(
+        "http://backend.genepinet.localdev:3000/v2/openapi.json"
```

Comment:
I think it would be better to use the health check endpoint rather than the openapi stuff since the health check is more closely part of our app's working structure and the openapi thing is more something provided by FastAPI.

`/v2/health`, see `health.py` in views

Reply:
yeah using the health endpoint seems better, also i think we're ok with just checking localdev, because if localdev is working that means that the backend container is able to serve requests (which should be the same case for staging, prod, etc). @jgadling what do you think?

Reply:
porqué no los dos?